client/test/config/steps/password.js

```diff
@@ -0,0 +1,92 @@
+'use strict';
+let assert = require('chai').assert;
+let sinon = require('sinon');
+
+
+describe('Step: password', () => {
+
+    let jQuery;
+    let jsdom;
```

Comment:
TODO: I should mock `configPasswordSubmit` not to call `jQuery.post`; therefor I should remove `jsdom`, and `jQuery` in these tests.